darts/tests/test_timeseries_static_covariates.py

```diff
@@ -104,6 +104,48 @@ def test_ts_from_x(self, tmpdir_module):
             ts, TimeSeries.from_json(ts_json, static_covariates=ts.static_covariates)
         )
 
+    def test_from_group_dataframe(self):
+        # checks that the time_index is of RangeIndex type when the time_col is a(n) (unsorted) list and/or a Rangeindex
+        group = ["a", "a", "a", "b", "b", "b"]
+        values = np.random.randn(len(group))
+
+        # for time as a unsorted list
+        time = [2, 1, 0, 0, 1, 2]
+        df = pd.DataFrame({
+            "group": group,
+            "time": time,
+            "x": values,
+        })
+        ts = TimeSeries.from_group_dataframe(df, group_cols="group", time_col="time")
+        assert isinstance(ts[0].time_index, pd.RangeIndex)
+        assert (ts[0].time_index == pd.RangeIndex(3)).all()
+        assert (ts[1].time_index == pd.RangeIndex(3)).all()
```

Comment:
You can write it like this everywhere (and remove the assert in the line above)
```suggestion
        assert ts[0].time_index.equals(pd.RangeIndex(3))
```